packages/expect/__typetests__/expect.test.ts

```diff
@@ -48,6 +48,7 @@ expectType<void>(
       expectType<Array<Error>>(this.suppressedErrors);
       expectType<string | undefined>(this.testPath);
       expectType<MatcherUtils>(this.utils);
+      expectType<Array<Tester>>(this.customTesters);
```

Comment:
Just a detail. Could you move this line above `this.dontThrow` assertion to keep alphabetical order?